engine/runtime/src/main/scala/org/enso/compiler/Compiler.scala

```diff
@@ -114,7 +114,7 @@ class Compiler(
     * @return an interpreter node whose execution corresponds to the top-level
```

Comment:
The `@return` description wasn't probably correct before and it also may need to be updated after the change - `CompilerResult` is a new type, right?